engine/classes/Elgg/DeprecationService.php

```diff
@@ -65,23 +60,10 @@ function sendNotice($msg, $dep_version, $backtrace_level = 1) {
 		$dep_major_version = (int)$dep_version_arr[0];
 		$dep_minor_version = (int)$dep_version_arr[1];
 
-		$visual = false;
```

Comment:
Annoys me too:
 - Long texts are difficult to read from error messages
 - If too many, some of them go outside the viewport (cannot even scroll because of the fixed position)
 - Copying function names is difficult, because the errors get closed when clicked